src/extensions/extensions.d.ts

```diff
@@ -5,8 +5,16 @@ export type ICIInfo = {
   repository_name: string
   repository_ref: string
   repository_commit_sha: string
+  branch_url: string
+  branch_name: string
+  pull_request: IPRInfo | boolean
```

Comment:
given the format of other properties, can we update the `pull_request` to `pull_request_name` and `pull_request_url`

Reply:
I had considered that as one of the option, the problem is with the way we have to determine if there is a PR for gitlab and others. However this felt like a better way to handle without many if conditions.

Let me revisit the changes and see if the logic can be simplified to accommodate the suggestion.

Reply:
@ASaiAnudeep Done